preloads.md

```diff
@@ -39,7 +42,8 @@ public override void Initialize(Dictionary<string, Dictionary<string, GameObject
    Object.DontDestroyOnLoad(BossPrefab);
    Object.DontDestroyOnLoad(CardPrefab);
 }
-``` 
+```
+
 You can then access your GameObject by doing `preloadedObjects[SceneName][GameObjectPath]`.
 
-> Note: Preloading objects increases the overall loading time for the game to start. If possible, you should preload the minimum number of objects that you need. 
+> Note: Preloading objects increases the overall loading time for the game to start. If possible, you should preload the minimum number of objects that you need.
```

Comment:
I know this is not new content but perhaps we should remark on reducing the number and size of scenes as well

Reply:
yea, was thinking of adding a line along the lines of "or pull your objects from small scenes that are easy to preload"